src/global.d.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 interface Document {
```

Comment:
why it's complaining though? any other way to fix this?

Reply:
No clue, but I think it's a bug. They get a lot of false positive issues lately: https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc++no-unused-vars